docs/Using-Fleet/REST-API.md

```diff
@@ -5170,6 +5185,12 @@ Returns a list of all queries in the Fleet instance.
     "name": "query1",
     "description": "query",
     "query": "SELECT * FROM osquery_info",
+    "team_id": null,
+    "interval": 3600,
+    "platform": "macos,windows,linux",
```

Comment:
Should be `darwin` not `macos`

(We use `darwin` internally for macOS.)

Reply:
@rachaelshaw Let me know if you are onboard, and I'll update this.

Reply:
@lucasmrod good catch 👍